docs/self-managed/operational-guides/backup-restore/webapps-backup.md

```diff
@@ -97,17 +107,55 @@ CAMUNDA_TASKLIST_BACKUP_REPOSITORYNAME=<es snapshot repository name>
 </TabItem>
 </Tabs>
 
+#### Optimize
+
+<Tabs groupId="config" className="tabs-hidden" defaultValue="yaml" values={
+[
+{label: 'YAML file', value: 'yaml', },
+{label: 'Environment variables', value: 'env', },
+]
+}>
+
+<TabItem value='yaml'>
+
+In file `environment-config.yaml`:
+
+```yaml
+backup:
+  repositoryName: <repository name>
+```
+
+</TabItem>
+<TabItem value='env'>
+
+```
+CAMUNDA_OPTIMIZE_BACKUP_REPOSITORY_NAME=<es snapshot repository name>
+```
+
+</TabItem>
+</Tabs>
+
+### Index prefix
+
+:::warning breaking change
+Since version 8.7 the `indexPrefix` of all webapps must match. By default it's set to `""`, but if it was overriden, then it must done consistently across Operate, Tasklist and Optimize.
+:::
+
 ## Create backup API
 
-During backup creation Operate can continue running. To create the backup, call the following endpoint:
+During backup creation webapps can continue running. To create the backup, call the following endpoint:
 
 ```
-POST actuator/backups
+POST actuator/backupHistory
```

Comment:
🙃 I guess it is a bit late for this, but why is it actually backupHistory? It is backing up more than history, right? It feels like it is not syntactically correct somehow 🤔

Reply:
Yes, but we never found a better term. I think part of the original proposals we had `backupZeebe` and `backupWebapps`